### ISSUE TITLE
List also local themes in lxqt-config-appearance

### ISSUE THREAD

**Issue by stefonarch** [lxqt/lxqt-config#1006]
Confronting[ gtk-conf](https://github.com/qtilities/gtk-conf) with "appearance" I noticed that the first lists and applies also themes in ~/.themes` directory.

##### Expected Behavior

Also local themes should be available in the dialog.
##### Current Behavior

Only installed themes are listed.
                                      
* Distribution & Version: all

**Comment by isf63** [comment]
I can confirm that `~/.local/share/themes/\<foo\>` GTK themes are detected but not `~/.themes/\<foo\>`.

Personally I've never used `~/.themes/` but it seems to be a GTK standard.

**Comment by tsujan** [comment]
This seems to me like an inconsistency in GTK. GNOME devs drop things as important as desktop icons or system tray but keep deprecated config places.

**Comment by stefonarch** [comment]
I made a search "install gtk themes manually" and the first 2 links (from 2021) say something like 
>GNOME GTK themes should be installed in the ~/.themes directory inside your home directory

**Comment by tsujan** [comment]
> Same for ~/.icons.... deprecated too afaik.

Yes,  such things should go into `$XDG_DATA_DIRS`. We shouldn't support "GNOME standard" vs. "standard", although GNOME devs may not care.

**Comment by isf63** [comment]
Can we use `~/.local/share/icons/default/index.theme` instead of `~/.icons/default/index.theme` in Appearance settings? (Hoping, to clean up the home folder)
